### PR TITLE
Allows the detective's and nuke op's holsters to be placed in the suit storage slot of armor vests

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
image
Sometimes you want to use your belt for something else anyways, or want multiple holsters, up to you really.

The chameleon traitor holster is left out of this because it can be changed to look like any belt item, which is more work to make function right than I care to do.

## Why It's Good For The Game
Allows the holsters to be used in different ways other than being put in a backpack or taking up the belt slot.

## Changelog
:cl:
balance: nuke op and detective holsters can now be placed in armor best suit storage
/:cl:

### CODE REVIEW

**Comment by imsxz** [comment]
this ruins the balance of the game

**Comment by SinfulBliss** [comment]
the holster was useless before since it was better to keep batong in belt as detective and throw gun in suit storage. now it has a use again. nice job

**Comment by necromanceranne** [comment]
Literally pointless keeping the holsters on your belt slot. You can just fill a secbelt with .38 and you can print them from the garden. If you're looking to transport ammo, you have alternatives in the same department you start in. The same goes for nukies, who really don't have a good reason to use them except maybe carrying their rocket launchers.

The holster is such a godawful item with the only notable feature was at one point actively jeopardizing the effectiveness of the shooter due to intercepting button presses to spin your gun like a fucking moron, which has no visual element at all so nobody would even goddamn notice it.

Good PR Pax, but make sure it doesn't cause dreaded pinksproite.

**Comment by Paxilmaniac** [comment]
> Good PR Pax, but make sure it doesn't cause dreaded pinksproite.

I already made sure it doesn't, how else am i supposed to look badass with two holsters at the same time if there are pink sprites

File: code/__DEFINES/inventory.dm [hunk 1]
```diff
@@ -207,6 +207,8 @@ GLOBAL_LIST_INIT(security_vest_allowed, typecacheof(list(
 	/obj/item/restraints/handcuffs,
 	/obj/item/tank/internals/emergency_oxygen,
 	/obj/item/tank/internals/plasmaman,
+	/obj/item/storage/belt/holster/detective,
+	/obj/item/storage/belt/holster/nukie
```
Comment (Mothblocks) [inline code/__DEFINES/inventory.dm:211]:
```suggestion
	/obj/item/storage/belt/holster/nukie,
```

**Comment by Mothblocks** [comment]
@Timberpoes you know, you might be a bit late on that

**Comment by Pumpkinoe** [comment]
Poor timber

**Comment by Technoturnovers** [comment]
you forgot to make this apply to the detective's armor vest, the armor vest the detective starts with... bruh.

**Pull request merged**